Ska/engarchive/fetch.py

```diff
@@ -1359,9 +1364,6 @@ def plot(self, *args, **kwargs):
                      **kwargs)
         plt.margins(0.02, 0.05)
 
-    def __len__(self):
-        return len(self.times)
-
```

Comment:
This change will change behavior. It is not just pep8/lint. Is it?

Reply:
This was a duplicate of the same definition elsewhere.